lib/cloudant.js

```diff
@@ -784,6 +786,15 @@ Cloudant.prototype.automigrate = function(models, cb) {
  *        "<key2>": -1
  *
 */
+
+Cloudant.prototype.getLimit = function(limit) {
+  return limit || this.getGlobalLimit();
+};
+
+Cloudant.prototype.getGlobalLimit = function() {
```

Comment:
Any reason for this to be a function and not just be a part of `getLimit`? 

```
Cloudant.prototype.getLimit = function(limit) {
  return limit || this.settings.globalLimit;
}
```

Reply:
It's more like a personal coding habit, I treat `getGlobalLimit` and `getLimit` as separate functionalities, e.g. you can add more approaches set the global limit value, currently it's from datasource settings only, but it may come from model or Env in the future.
And for `getLimit`, we may add options and corresponding logic.

Reply:
Fair enough. My personal preference is just having one function :P 

This isn't a blocker.